src/mumble/Database.cpp

```diff
@@ -47,7 +48,11 @@ bool Database::findOrCreateDatabase() {
 	datapaths << Global::get().qdBasePath.absolutePath();
 	datapaths << QStandardPaths::writableLocation(QStandardPaths::DataLocation);
 #if defined(Q_OS_UNIX) && !defined(Q_OS_MAC)
-	datapaths << QDir::homePath() + QLatin1String("/.config/Mumble");
+	QString configDirectory = EnvUtils::getenv(QLatin1String("XDG_CONFIG_HOME")) + QLatin1String("/Mumble");
+	if(configDirectory.isEmpty()) {
```

Comment:
How would ``configDirectory`` ever be empty at this point? The code is adding ``/Mumble`` to the result of ``getenv`` no matter what, right?